packages/cuba-react-ui/src/ui/table/DataTable.tsx

```diff
@@ -29,6 +29,9 @@ import {
 
 // todo may be it make sense to add 'defaultDataTableProps' const instead of describe each property in jsdoc,
 //  it's easy to forget update js docs after DataTable.defaultProps changes
```

Comment:
I think this TODO could be removed. Now we have comprehensive description for each property.